spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe "the ingredients index" do
+  it "see a list of all the ingredients including their name and cost" do
+    beef = Ingredient.create(name: "Ground Beef", cost: 2)
+    cheese = Ingredient.create(name: "American Cheese", cost: 1)
+    bun = Ingredient.create(name: "Brioche Bun", cost: 3)
+
+    visit "/ingredients"
+
+    expect(page).to have_content(beef.name)
```

Comment:
This test is good, but could be more robust with within blocks. For example, `expect(page).to have_content(beef.cost)` is just expecting the number 2 to be somewhere on the page. What if there was a phone number on the page that contained a 2? That could give a false positive. Within blocks are handy so that you can narrow down where you want to see the 2 (i.e. in the section of the page just for beef)

Another option is to expect the page to have `"Cost: #{beef.cost}"`